this has been packaged as /usr/bin/uutils since the inception of this aport,
early 2021, let's not break that expectation

--- a/src/bin/coreutils.rs
+++ b/src/bin/coreutils.rs
@@ -53,7 +53,7 @@ fn main() {
     // binary name ends with util name?
     let matched_util = utils
         .keys()
-        .filter(|&&u| binary_as_util.ends_with(u) && !binary_as_util.ends_with("coreutils"))
+        .filter(|&&u| binary_as_util.ends_with(u) && !binary_as_util.ends_with("coreutils") && !binary_as_util.ends_with("uutils"))
         .max_by_key(|u| u.len()); //Prefer stty more than tty. coreutils is not ls
 
     let util_name = if let Some(&util) = matched_util {
